.github/workflows/pypi-publish.yml

```diff
@@ -29,15 +29,6 @@ jobs:
     - name: Verify the distributions
       run: twine check dist/*
 
-    - name: Publish distribution to Test PyPI
-      # The following upload action cannot be executed in the forked repository.
-      if: github.repository == 'optuna/optuna'
-      uses: pypa/gh-action-pypi-publish@v1.1.0
-      with:
-        user: __token__
-        password: ${{ secrets.test_pypi_password }}
-        repository_url: https://test.pypi.org/legacy/
-
```

Comment:
How about switching testpypi and pypi depending on the build trigger? Currently, we need to synchronize two files and we may not be able to find configuration errors due to synchronization errors.